src/app/public/modules/vertical-tabset/vertical-tabset.component.spec.ts

```diff
@@ -108,12 +113,37 @@ describe('Vertical tabset component', () => {
     fixture.detectChanges();
     let el = fixture.nativeElement;
 
+    // check open tab
+    const visibleTabs1 = getVisibleTabContent(fixture);
+    expect(visibleTabs1.length).toBe(1);
+    expect(visibleTabs1[0].textContent.trim()).toBe('Group 1 Tab 1 content');
+
+    // check order of vertical tab content
+    const allTabContentElements = fixture.nativeElement.querySelectorAll('.sky-vertical-tab-content-pane');
+    expect(allTabContentElements.length).toBe(2);
+    expect(allTabContentElements[0].textContent.trim()).toBe('Group 1 Tab 2 content');
+    expect(allTabContentElements[1].textContent.trim()).toBe('Group 1 Tab 1 content');
+
     // open second group
     const group = el.querySelectorAll('.sky-vertical-tabset-group-header');
     group[1].click();
 
     fixture.detectChanges();
 
+    // check open tab
+    const visibleTabs2 = getVisibleTabContent(fixture);
```

Comment:
Could we just turn `visibleTabs1` to `visibleTabs` and a `let` instead of a `const` and just reassign it oin line 134, etc.?

Same for `allTabsContentElements1` and etc.

It doesn't appear we are using the earlier value any more so I think we are fine to reassign.

Reply:
Yep, I'll clean these up.